doc/statistics/univariate_statistics.qbk

```diff
@@ -121,12 +212,17 @@ In general, you can store your data in an Eigen array, and Armadillo vector, `st
 These routines are usable in float, double, long double, and Boost.Multiprecision precision, as well as their complex extensions whenever the computation is well-defined.
 For certain operations (total variation, for example) integer inputs are supported.
 
+/Nota bene/: The default execution policy for every function is std::execution::seq.
+The use of parallel execution policies (std::execution::par or par_unseq) generally doubles the error introduced in the computation, but is still minimal.
```

Comment:
Might be good to say "Parallel routines must use random access containers, not just forward iterable."